### DISCUSSION TITLE
SP111 v1.1 switchs off randomly

### DISCUSSION THREAD

**Discussion by DandeMC** [arendst/Tasmota#15644] [Q&A] [answered]
Hi,

my gosund smart plug SP111 v1.1 switchs off randomly and I don't now why. It's on version 11.1.0 (the problem was on 11.0 too). How can I check this?

**Comment by sfromis** [comment, marked as answer]
https://tasmota.github.io/docs/FAQ/#my-device-randomly-switches-on-and-off-do-i-have-ghosts-in-my-house

Watch the console as it happens, after having set `Weblog 3`.
